core/runtime/jsruntime.rs

```diff
@@ -305,9 +305,6 @@ pub type CompiledWasmModuleStore = CrossIsolateStore<v8::CompiledWasmModule>;
 pub struct JsRuntimeState {
   known_realms: Vec<JsRealmInner>,
   pub(crate) has_tick_scheduled: bool,
-  /// A counter used to delay our dynamic import deadlock detection by one spin
```

Comment:
This lives in ModuleMap for now, but it will likely move around a bit as we refactor.